### PR TITLE
tuto(serverless) - update tuto

### PR DESCRIPTION
**Pull request by lucasscw**
- Added:
   - Full code at the end of the tutorial
   - Check Typos and errors

### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 1]
```diff
@@ -40,12 +39,12 @@ This project contains two functions where:
 
 ## Creating the BucketScan function
 
-1. Set the following environment variables:
-    * `SOURCE_BUCKET`: Key of your source bucket
+1. Set the following Environment Vairables at the function level:
```
Comment (bene2k1) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:42]:
```suggestion
1. Set the following environment variables at the function level:
```

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 2]
```diff
@@ -92,47 +91,60 @@ This project contains two functions where:
...
+          };
+        }
+      });
+    };
     ```
 
 ## Creating the ImageTransformation function
 
-The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.com/lambda/latest/dg/with-s3-tutorial.html#with-s3-tutorial-create-function-code) tutorial.
+The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.com/lambda/latest/dg/with-s3-tutorial.html#with-s3-tutorial-create-function-code) tutorial. We recommand using the last version of Node.
```
Comment (bene2k1) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:136]:
```suggestion
The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.com/lambda/latest/dg/with-s3-tutorial.html#with-s3-tutorial-create-function-code) tutorial. We recommend using the latest version of Node for this tutorial.
```

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 3]
```diff
@@ -92,47 +91,60 @@ This project contains two functions where:
...
 
 1. Install `sharp` using the fbs `linuxmusl` platform to ensure compatibility with Scaleway’s node runtime:
     ```bash
     npm install --platform=linuxmusl --arch=x64 sharp --ignore-script=false --save
     ```
 
-2. Set the following environment variables: 
-    * `SOURCE_BUCKET`: Key of your source bucket
+2. Set the following Environment Variables at the function level:
```
Comment (bene2k1) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:143]:
```suggestion
2. Set the following environment variables at the function level: 
```

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 4]
```diff
@@ -232,21 +264,24 @@ The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.co
...
+          "Content-Type": "application/json"
+        }
       };
     };
     ```
 
 ## Pushing the two functions to Scaleway Serverless 
 
 Please refer to the Serverless Documentation to learn how to package and deploy your functions using [Serverless Framework](/compute/functions/api-cli/fun-uploading-with-serverless-framework/) or a [zip-file](/compute/functions/how-to/package-function-in-zip/).
+Once the ImageTransform function is deployed copy its URL (from the Settings Tab in the console or in the Serverless Framework logs) into the BucketScan environment Varible.
```
Comment (bene2k1) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:279]:
```suggestion
Once the `ImageTransform` function is deployed, copy its URL (from the settings tab in the console or in the Serverless Framework logs) into the `BucketScan` environment variable. 
```

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 5]
```diff
@@ -232,21 +264,24 @@ The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.co
...
+        }
       };
     };
     ```
 
 ## Pushing the two functions to Scaleway Serverless 
 
 Please refer to the Serverless Documentation to learn how to package and deploy your functions using [Serverless Framework](/compute/functions/api-cli/fun-uploading-with-serverless-framework/) or a [zip-file](/compute/functions/how-to/package-function-in-zip/).
+Once the ImageTransform function is deployed copy its URL (from the Settings Tab in the console or in the Serverless Framework logs) into the BucketScan environment Varible. 
+Once everything is well deployed, test the BucketSan functions using its URL
```
Comment (RoRoJ) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:280]:
```suggestion
Once everything is deployed, test the BucketSan functions using its URL.
```

**Review by bene2k1 — APPROVED** [review]

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 6]
```diff
@@ -31,7 +31,6 @@ This project contains two functions where:
 
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have created a [Serverless Function namespace](/compute/functions/how-to/create-a-functions-namespace/)
-  - You have downloaded the corresponding [kubeconfig](/compute/kubernetes/how-to/edit-cluster/) file and [kubectl](/compute/kubernetes/how-to/connect-cluster-kubectl/) is configured and working
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply tutorials/transform-images-using-faas-and-nodejs/index.mdx:34]: ligne supprimé

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**